core/src/main/java/org/testcontainers/containers/ContainerState.java

```diff
@@ -43,8 +43,18 @@
      * Get the IP address that this container may be reached on (may not be the local machine).
      *
      * @return an IP address
+     * @see #getHost()
      */
     default String getContainerIpAddress() {
```

Comment:
Shouldn't we add `@Deprecated` here?

Reply:
Maybe let's not deprecate it yet, since it is widely used and there is no strong reason for deprecation (aside from the naming PoV)

Once `getHost` is widely adopted by the community, we can deprecate `getContainerIpAddress`